Makefile

```diff
@@ -0,0 +1,11 @@
+#shellcheck disable=code
+.PHONY: test-and-migrate migrate test
+
+# Example: make test FILTER=tests/Jobs/ElasticSearchIndexInitTest.php
+test:
+	docker-compose exec api vendor/bin/phpunit ${FILTER}
+
+migrate:
```

Comment:
Hmm, `migrate` isn't an ideal name here IMO. Running laravel migrations is just `php artisan migrate`, and this long command here is more of an initial db setup kind of thing?

Reply:
+1

Reply:
`init` instead?